testsuite/flattening/modelica/scodeinst/Makefile

```diff
@@ -662,6 +662,9 @@ FunctionMultiOutput4.mo \
 FunctionMultiOutput5.mo \
 FunctionNoOutput1.mo \
 FunctionNonInputOutputParameter.mo \
+FunctionPartialDerivative1.mo \
+FunctionPartialDerivative2.mo \
+FunctionPartialDerivative3.mo \
```

Comment:
I think you forgot to add the actual models. :slightly_smiling_face:

Reply:
Yep :smile: